ts/src/base/Exchange.ts

```diff
@@ -1358,6 +1358,18 @@ export default class Exchange {
     // ------------------------------------------------------------------------
     // METHODS BELOW THIS LINE ARE TRANSPILED FROM JAVASCRIPT TO PYTHON AND PHP
 
+    findMessageHashes (client, element: string): string[] {
```

Comment:
@pcriadoperez there is a lengthy discussion regarding the best way to handle `watchTickers`. I also believe this is the easiest way of making it right without needing an dispatcher/subscription-manager. The only downside I see is the necessity of doing a lot of string comparisons (with the `indexOf`), but in practice and unless the user specifies hundreds of symbols it should be ok I guess.

Reply:
(actually I did something similar in `whitebit` but was eventually removed)

Reply:
Yes, I think that could be the only issue, but I believe it should be fast as it's stored in memory.
If we wanted to get around it maybe a more efficient way would be to store a hashmap that indexes the futures by symbol. But then we would be introducing another object and more complexity, and not sure if it's worth it for this case.

Reply:
@pcriadoperez Yeah let's keep it as is for now